<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoverageOverlayDockWidget</class>
 <widget class="QDockWidget" name="CoverageOverlayDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>523</height>
   </rect>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>覆盖层叠加</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0,4">
    <item>
     <layout class="QGridLayout" name="mButtonGridLayout" columnstretch="1,1,1">
      <item row="0" column="2">
       <widget class="QPushButton" name="mUnloadDataButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>卸载所选文件</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="mLoadDataButton">
        <property name="text">
         <string>加载覆盖文件</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="mLoadUnloadSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="mLoadedDataList">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="mCoverageDataGroup">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QComboBox" name="mFieldSelector"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="mFieldLabel">
         <property name="text">
          <string>测量:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="mTypeValue">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="mTypeLabel">
         <property name="text">
          <string>类型:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="mSubTypeLabel">
         <property name="text">
          <string>子类型:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="mSubtypeValue">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="mRangeGroup">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>范围</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="mAutoScaleCheck">
         <property name="text">
          <string>自动</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="mMinimumLabel">
         <property name="text">
          <string>最小:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="mMinimumEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="mMaximumLabel">
         <property name="text">
          <string>最大:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="mMaximumEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="mDisplayGroup">
      <property name="title">
       <string>显示选项</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <item>
        <layout class="QGridLayout" name="mDisplayControlsLayout" columnstretch="0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="mOpacityLabel">
           <property name="text">
            <string>不透明度:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="mOpacitySlider">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="pageStep">
            <number>32</number>
           </property>
           <property name="value">
            <number>127</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="mGradientLabel">
           <property name="text">
            <string>渐变类型:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="mGradientSelector"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="mLegendLayout" stretch="0,1">
         <item>
          <widget class="UtQtGradientLegend" name="mGradientLegend" native="true"/>
         </item>
         <item>
          <spacer name="mLegentSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>195</width>
             <height>188</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtGradientLegend</class>
   <extends>QWidget</extends>
   <header>UtQtGradientLegend.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mLoadDataButton</tabstop>
  <tabstop>mUnloadDataButton</tabstop>
  <tabstop>mLoadedDataList</tabstop>
  <tabstop>mFieldSelector</tabstop>
  <tabstop>mTypeValue</tabstop>
  <tabstop>mSubtypeValue</tabstop>
  <tabstop>mAutoScaleCheck</tabstop>
  <tabstop>mMinimumEdit</tabstop>
  <tabstop>mMaximumEdit</tabstop>
  <tabstop>mGradientSelector</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
